src/inmanta/loader.py

```diff
@@ -182,36 +182,6 @@ def __init__(self, code_dir: str) -> None:
         self.__modules: Dict[str, Tuple[str, types.ModuleType]] = {}  # A map with all modules we loaded, and its hv
 
         self.__check_dir()
-        self.load_modules()
-
-    def load_modules(self) -> None:
-        """
-        Load all existing modules
-        """
-        mod_dir = os.path.join(self.__code_dir, MODULE_DIR)
-        configure_module_finder([mod_dir])
```

Comment:
We should move this line into the constructor.

Reply:
Thx!